core/src/crds_gossip_pull.rs

```diff
@@ -117,6 +119,7 @@ pub struct CrdsGossipPull {
     /// hash and insert time
     purged_values: VecDeque<(Hash, u64)>,
     pub crds_timeout: u64,
+    pub msg_timeout: u64,
```

Comment:
CrdsGossipPull doesn't go over the wire right?

Reply:
Nope.